test/unit/docstrings_test.py

```diff
@@ -0,0 +1,99 @@
+import inspect
+import pkgutil
+import pytest
+import re
+
+import kiwi
+
+
+def members(module, predicate=None):
+    """Yields all functions of a module if predicate=None
+
+    :param module: the given module to search for members
+    :type module: :class:`module`
+    :param predicate: a function with expects an object and returns True or False
+                      see :func:`inspect.isfunction` as an example
+    :type prediate: :class:`function`
+    """
+    predicate = inspect.isfunction if predicate is None else predicate
+    for _, func, in inspect.getmembers(module, predicate):
+        if func.__module__ == module.__name__:
+            yield func
+
+
+def getmodules(module, predicate=None):
+    """Yields recursively sub module names from a given module
+
+    :param module: the given module to search for members
+    :type module: :class:`module`
+    :param predicate: a function with expects an object and returns True or False
+                      see :func:`inspect.isfunction` as an example
+    :type prediate: :class:`function`
+    """
+    prefix = module.__name__ + "."
+    for importer, modname, ispkg in pkgutil.iter_modules(module.__path__, prefix):
+        module = importer.find_module(modname).load_module(modname)
+        yield from members(module, predicate)
```

Comment:
This is most probably conflicting with the loading pytest does

In addition there are doc strings evaluated by docopt. that covers all code below tasks/

Sorry I have no idea at the moment how to solve this